modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -639,6 +639,11 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
       */
     CV_EXPORTS_W Net readNetFromTensorflow(const String &model, const String &config = String());
 
+    /** @brief Reads a network model stored in Tensorflow model in memory
+     */
+    CV_EXPORTS_W Net readNetFromTensorflowBuffer(const char *bufferModel, size_t lenModel,
```

Comment:
Let's keep the same name for it, `readNetFromTensorflow`.

I think we can use brief description like at https://docs.opencv.org/master/d3/d63/classcv_1_1Mat.html#a2ec3402f7d165ca34c7fd6e8498a62ca: `This is an overloaded member function...` where only parameters are described.

Have you tested python / Java bindings for it?